code_schemes/s02_dadaab_impact_made.json

```diff
@@ -3,6 +3,38 @@
   "Name": "S02 DADAAB Impact Made",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "CodeID": "code-22aad8d4",
+      "CodeType": "Normal",
+      "DisplayText": "There are changes",
+      "NumericValue": 1,
+      "StringValue": "There_are_changes",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-72691710",
+      "CodeType": "Normal",
+      "DisplayText": "Access to encouragement",
+      "NumericValue": 2,
+      "StringValue": "Access_to_encouragement",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-44f81b2d",
+      "CodeType": "Normal",
+      "DisplayText": "Gender equality",
+      "NumericValue": 3,
+      "StringValue": "Gender_equality",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-dd264c5c",
+      "CodeType": "Normal",
+      "DisplayText": "other",
```

Comment:
```suggestion
      "DisplayText": "other theme",
```

Reply:
we have another `other` so the pipeline will pick this as a duplicate. Rename to other theme

Reply:
updated thanks